modules/cnf-deploying-the-numa-aware-scheduler.adoc

```diff
@@ -90,6 +90,30 @@ spec:
 $ oc create -f nro-machineconfig.yaml
 ----
 
+. Create the `NUMAResourcesOperator` resource that reads the `ResourceTopologyExporter` configuration from the `KubeletConfig` configured on the worker nodes:
+
+.. Save the following YAML in the `nrop.yaml` file:
++
+[source,yaml]
+----
+apiVersion: nodetopology.openshift.io/v1alpha1
+kind: NUMAResourcesOperator
+metadata:
+  name: numaresourcesoperator
+spec:
+  nodeGroups:
+  - machineConfigPoolSelector:
+      matchLabels:
+        pools.operator.machineconfiguration.openshift.io/worker: ""
+----
+
+.. Create the `NUMAResourcesOperator` CR:
++
+[source,yaml]
+----
+$ oc create -f nrop.yaml
+----
+
```

Comment:
an example output would be as follows:

```
numaresourcesoperator.nodetopology.openshift.io/numaresourcesoperator created
```

and one can display the resources as follows:

```
oc get numaresourcesoperators.nodetopology.openshift.io 
NAME                    AGE
numaresourcesoperator   45h
```